docs/CONFIG.md

```diff
@@ -9243,6 +9308,33 @@ TipCapMinimum is the minimum gas tip to use when submitting transactions to the
 
 (Only applies to EIP-1559 transactions)
 
+## EVM.GasEstimator.DAOracle
+```toml
+[EVM.GasEstimator.DAOracle]
+OracleType = 'OPOracle' # Example
+OracleAddress = '0x420000000000000000000000000000000000000F' # Example
+CustomGasPriceAPICalldata = '' # Default
+```
+
+
+### OracleType
+```toml
+OracleType = 'OPOracle' # Example
+```
+OracleType refers to the oracle family this config belongs to (OP, Arbitrum, etc).
+
+### OracleAddress
+```toml
+OracleAddress = '0x420000000000000000000000000000000000000F' # Example
+```
+OracleAddress is the address of the oracle contract.
+
+### CustomGasPriceAPICalldata
+```toml
+CustomGasPriceAPICalldata = '' # Default
+```
+CustomGasPriceAPICalldata is optional and can be set to call a custom gas price API at the given OracleAddress.
```

Comment:
Not sure I understand this fully, what does an API mean here? means a function signature? Also could we please add an example from a real scenario?

Reply:
will rename this to `CustomGasPriceCalldata`. Chatted with @matYang , there isn't currently a chain with one of these custom gas oracle functions but we expect to see it relatively soon and would like to maintain the flexibility. 

For now I'm going to remove support for this custom calldata logic from the OP oracle and create a separate `CustomCalldataOracle` to keep the OP oracle logic a bit cleaner

Reply:
sounds good thank you